tools/pontoon/renaming_a_file.md

```diff
@@ -0,0 +1,15 @@
+# Renaming a localization file in a project without losing data
+
+In some cases, project owners will want to rename a file for clarity, or for technical reasons. If it’s only done on the code side, it will unfortunately require the file to be re-translated from scratch. Since this is a special case, we can work around this and rename the file in Pontoon while keeping existing data (translations, suggestions, attribution, history). The process is not complicated, but it has to be highly coordinated.
+
+## Get a Pull Request ready
+
+Either you or the project owners will get a Pull Request ready that renames the file for all locales. Using a script is recommended as you will need to generate the patch not too long before the next steps to avoid conflicts. Make sure the patch is only renaming the file and not touching content. If you need to move strings into another file, you can do it in a separate Pull Request and merge it after the renaming has been synced.
+
+## Rename the resource in Pontoon admin panel
+
+Wait for the current Pontoon sync cycle to end, and make sure you are already logged into Pontoon admin, then merge the Pull Request.
```

Comment:
Line breaks are not effective in Markdown. If you need one, you need a `<br>` or much worse methods (two empty spaces at the end of the line, which obviously any editor will strip…).

In this specific case, I think you want 3 paragraphs.